addons/ai/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,23 @@
+class Extended_PreStart_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_FILE(XEH_preStart));
+    };
+};
+
+class Extended_PreInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_FILE(XEH_preInit));
+    };
+};
+
+class Extended_PostInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_FILE(XEH_postInit));
+    };
+};
+
+class Extended_FiredNear_EventHandlers {
+    class CAManBase {
+        init = QUOTE(call COMPILE_FILE(mortarSupportFiredNear));
```

Comment:
Should use ```FUNC(mortarSupportFiredNear)``` here.
See https://github.com/uksf/Modpack/blob/master/addons/apache/CfgEventHandlers.hpp#L16 for example